meinberlin/apps/budgeting/assets/__tests__/ListItemBadges.jest.jsx

```diff
@@ -3,59 +3,29 @@ import { render, screen } from '@testing-library/react'
 import { ListItemBadges } from '../ListItemBadges'
 
 // testing data:
-const categoryBadge = [{
-  type: 'category',
-  value: { id: 0, name: 'category1' }
-}]
-
-const labelsBadges = [
-  {
-    type: 'label',
-    value: { id: 0, name: 'label1' }
-  },
-  {
-    type: 'label',
-    value: { id: 1, name: 'label2' }
-  },
-  {
-    type: 'label',
-    value: { id: 2, name: 'label3' }
-  }
+const someBadges = [
+  ['category', 'category1'],
+  ['label', 'label1'],
+  ['label', 'label2']
 ]
 
-const pointLabelBadge = [{
-  type: 'pointLabel',
-  value: 'labelwithicon'
-}]
-
-const budgetBadge = [{
-  type: 'budget',
-  value: '20000'
-}]
-
-const modFeedbackBadge = [{
-  type: 'modFeedback',
-  value: ['CONSIDERATION', 'Under consideration']
-}]
+const pointLabelBadge = [
+  ['point_label', 'labelwithicon']
+]
 
-test('displaying category badge', () => {
-  render(
-    <ListItemBadges
-      badges={categoryBadge}
-    />
-  )
-  expect(screen.getByText('category1')).toBeTruthy()
-})
```

Comment:
I was a bit sad to delete tests, but as the component got a lot dumber with the serializer having all the info, they didn't make sense anymore. When I got it right, it only tests if the badges are there, right? We could still test if the moderator_statement is yellow and if the point_label has the icon or do the tests magically do that? @khamui Maybe you could have a look? (I will merge if the tests are actually all fine this time, though. :sweat_smile: )